api/credential_test.go

```diff
@@ -25,7 +25,11 @@ import (
 
 func TestCredentialsHandler(t *testing.T) {
 	conf := internal.NewConfigurationWithDefaults(configx.SkipValidation())
+	conf.SetForTest(t, configuration.MutatorIDTokenIsEnabled, true)
```

Comment:
All these are required because now only valid rules are considered by the rule repository.